cmd/minikube/cmd/config/kubernetes_version.go

```diff
@@ -34,3 +38,29 @@ func supportedKubernetesVersions() (releases []string) {
 	}
 	return releases
 }
+
+// GetGithubKubernetesVersions returns reverse-sort Kubernetes releases
+func GetGithubKubernetesVersions(ver string) ([]string, error) {
+	ghc := github.NewClient(nil)
+
+	opts := &github.ListOptions{PerPage: 100}
+	var releases []string
+	for {
+		rls, resp, err := ghc.Repositories.ListReleases(context.Background(), "kubernetes", "kubernetes", opts)
+		if err != nil {
+			return nil, err
+		}
+		for _, rl := range rls {
+			tag := rl.GetTagName()
+			if semver.IsValid(tag) {
+				releases = append(releases, tag)
```

Comment:
Actually, this could be made super efficient by simply calling `GetReleaseByTag`

https://pkg.go.dev/github.com/google/go-github/v53/github#RepositoriesService.GetReleaseByTag

https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-a-release-by-tag-name